flexget/plugins/sites/site_1337x.py

```diff
@@ -63,12 +66,12 @@ def url_rewrite(self, task, entry):
 
         soup = get_soup(page.content)
 
-        magnetURL = str(soup.find('a',id='magnetdl').get('href')).lower()
-        torrentURL = str(soup.find('a',id='torrentdl').get('href')).lower()
+        magneturl = str(soup.find('a', id='magnetdl').get('href')).lower()
```

Comment:
to be semantically pep8 compliant it should probably be `magnet_url` and `torrent_url`